code/src/components/App.js

```diff
@@ -0,0 +1,55 @@
+import React, { useState, useEffect } from 'react';
+import { HappyForm } from './HappyForm';
+import { HappyThought } from './HappyThought';
+
+export const App = () => {
+  const [happyThoughts, setHappyThoughts] = useState([]);
+
+  const happyThoughtsURL =
+    'https://happy-thoughts-technigo.herokuapp.com/thoughts';
```

Comment:
maybe you can store the url in a separate component to make it more clean? (like maks did)